sdk/inc/azure/core/az_log.h

```diff
@@ -61,22 +65,21 @@ typedef enum
 } az_log_classification;
 
 /**
- * @brief az_log_message_fn defines the signature of the callback function that application
- developers
- * must write in order to receive Azure SDK log messages.
+ * @brief Defines the signature of the callback function that application developers must write in
+ * order to receive Azure SDK log messages.
  *
- * @param classification The log message's az_log_classification.
- * @param message The log message.
-*/
+ * @param[in] classification The log message's #az_log_classification.
+ * @param[in] message The log message.
+ */
 typedef void (*az_log_message_fn)(az_log_classification classification, az_span message);
 
 /**
- * @brief az_log_set_classifications allows the application to specify which
- * az_log_classification types it is interested in receiving. If no classifications are set
- * (NULL), the application will receive log messages for all az_log_classification values.
+ * @brief Allows the application to specify which #az_log_classification types it is interested in
+ * receiving. If no classifications are set (`NULL`), the application will receive log messages for
+ * all #az_log_classification values.
  *
- * @param classifications An array of az_log_classification values.
- *                        The last element of the array must be AZ_LOG_END_OF_LIST.
+ * @param[in] classifications __[nullable]__ An array of az_log_classification values, terminated by
```

Comment:
What does nullable mean here? Consider moving the sentence from the brief  to the parameter description.